sdk/monitor/azure-monitor-query/azure/monitor/query/_logs_query_client.py

```diff
@@ -31,21 +31,21 @@ class LogsQueryClient(object):  # pylint: disable=client-accepts-api-version-key
     apps can be consolidated into a single Azure Log Analytics workspace.
 
     The various data types can be analyzed together using the
-    [Kusto Query Language](https://docs.microsoft.com/azure/data-explorer/kusto/query/)
-
-    .. admonition:: Example:
-
-    .. literalinclude:: ../samples/sample_single_logs_query.py
-        :start-after: [START client_auth_with_token_cred]
-        :end-before: [END client_auth_with_token_cred]
-        :language: python
-        :dedent: 0
-        :caption: Creating the LogsQueryClient with a TokenCredential.
+    [Kusto Query Language](https://learn.microsoft.com/azure/data-explorer/kusto/query/)
 
     :param credential: The credential to authenticate the client.
     :type credential: ~azure.core.credentials.TokenCredential
     :keyword endpoint: The endpoint to connect to. Defaults to 'https://api.loganalytics.io/v1'.
     :paramtype endpoint: Optional[str]
+
+    .. admonition:: Example:
+
+        .. literalinclude:: ../samples/code_snippets.py
```

Comment:
Would it make sense to call the file something like "authenticate" or "build" client.py? IMO if it lives under the sample directory it should be geared towards some scenario

Reply:
Yea, good point. I think having these snippets be more specific would be more appropriate. I'll make dedicated "authentication" samples for sync and async clients.

Reply:
looks good!